ExampleApp/ExampleApp/Screens/CentralServiceChacacteristics/CharacteristicsViewController.swift

```diff
@@ -0,0 +1,85 @@
+import RxBluetoothKit
+import RxSwift
+import UIKit
+
+class CharacteristicsViewController: UITableViewController {
+
+    init(characteristics: [Characteristic], bluetoothProvider: BluetoothProvider) {
+        self.characteristics = characteristics
+        self.bluetoothProvider = bluetoothProvider
+        super.init(nibName: nil, bundle: nil)
+
+        navigationItem.title = "Characteristics"
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    // MARK: - View
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.register(CharacteristicCell.self, forCellReuseIdentifier: CharacteristicCell.reuseId)
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
```

Comment:
I think you can remove this method because you are not calling any new code.

Reply:
Of course I can 🤦‍♂️ :D